### PR TITLE
fix: show test keys in dev environment

### LINKED ISSUE #7984: Show test/live key of Stripe under Payments section
**Describe the bug**
Show test/live key of Stripe under Payments section depending on the environment being used, if  local/test.eventyay.com is used - show the test keys else show live keys. Currently, the live keys are shown in both environments.

image

### PR DESCRIPTION
**Pull request by progmatic-99**
Fixes #7984

### CODE REVIEW

**Comment by progmatic-99** [comment]
Screenshot 2022-01-20 at 23-46-09 Payment Gateway Settings Administration eventyay

File: app/components/forms/admin/settings/payment-gateway-form.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ export default Component.extend(FormMixin, {
   }),
 
   stripeMode: computed(function() {
-    return ENV.environment === 'development' || ENV.environment === 'test' ? 'debug' : 'production';
+    return ENV.environment === 'production' ? 'production' : 'debug';
```
Comment (pc-beast) [inline app/components/forms/admin/settings/payment-gateway-form.js:209]: It will not let us change the modes from UI

**Review by pc-beast — COMMENTED** [review]
UI always changes to test mode in localhost when i save and reload the live mode radio.

**Review by pc-beast — COMMENTED** [review]
Same issue. doesn't work.

**Comment by mariobehling** [comment]
What is the status of this long standing PR?

**Comment by mariobehling** [comment]
Ok, closing this PR as it does not seem to work, right?

**Pull request closed without merging**